profiles/enn-dee.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/89350768/173366815-a919027a-2c53-4b9d-a029-ef0dbb2ede27.png",
```

It looks like the image link is different from the issue. Do you want to change it?
```suggestion
  "image": "https://user-images.githubusercontent.com/89350768/173365612-f5ddec29-c3f8-4bc4-a616-d9d0fa2ea6fd.png",
```